test/units/cli/test_adhoc.py

```diff
@@ -105,7 +105,9 @@ def test_ansible_version(capsys, mocker):
         version_lines = version[0].splitlines()
 
     assert len(version_lines) == 9, 'Incorrect number of lines in "ansible --version" output'
-    assert re.match(r'ansible \[core [0-9.a-z]+\]$', version_lines[0]), 'Incorrect ansible version line in "ansible --version" output'
+    assert re.match(r'ansible \[core [0-9.a-z]+\]'
```

Comment:
Instead of checking the entire version string, lets just remove the `$` from the regex.